cosmoz-page-router.js

```diff
@@ -472,6 +474,11 @@ class CosmozPageRouter extends PolymerElement {
 				this._fireEvent('import-error', importErrorEvent);
 			};
 
+		if (importUri.endsWith('.js')) {
+			this._fireEvent('route-loading', eventDetail);
+			import(importUri).catch(importErrorCallback).then(importLoadedCallback);
```

Comment:
Switching the callbacks around will also catch errors thrown in the `importLoadedCallback`.
`import(importUri).then(importLoadedCallback).catch(importErrorCallback);`

Reply:
True, but it would be a different behavior from html-based imports.
I think cosmoz-page-router should drop the html-based imports and just focus on a much slimmer implementation.

This PR is mainly to get the basic support in so we can get further with FE.